lib/samwise/client.rb

```diff
@@ -30,30 +30,25 @@ def excluded?(duns: nil)
       JSON.parse(response.body)["hasKnownExclusion"] == false
     end
 
-    def small_business?(duns: nil, naicsCode: nil)
+    def small_business?(duns: nil)
       response = lookup_duns(duns: duns)
       data = JSON.parse(response.body)["sam_data"]["registration"]
 
-      if data["certifications"] == nil
-        return false
+      far_responses = data['certifications']['farResponses']
+      response_to_small_biz = far_responses.find do |r|
```

Comment:
can we do a more descriptive var name than `r`? I generally try to stay away from single-letter vars where possible